KimYeSeung/210930/Scripts/BGM.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class BGM : MonoBehaviour
+{
+    public AudioClip BGM1;
+    public AudioClip BGM2;
+    public AudioClip BGM3;
+    AudioSource audioSource;
+    public playermove Player;
+    public GameObject player;
+
+    Rigidbody2D rigid;
+    
+    void Awake() {
+        player = GameObject.Find("player");
```

Comment:
```suggestion
        player = Transform.FindWithTag("player").gameObject;
```

단순히 오브젝트의 이름만으로 player가 어떤 아이인지 찾아낼 수 있나요? `Instantiate` 메소드를 통해 생성된 오브젝트의 경우, 오브젝트 이름에 `"(clone)"` 이라는 추가적인 이름이 붙는다는 것 알고 계셨나요?
그러한 경우에는 플레이어를 어떻게 찾을 수 있을까요?

또, player는 public 필드라서 유니티 에디터 내에서도 할당해줄 수 있는 상태인데, 이미 맵에 생성되어 있는 player를 넣어주는 거라면 굳이 `GameObject.Find` 를 통해서 부하를 주는 것 보다 애시당초 에디터에서 넣어주는 방식이 훨씬 가볍지 않을까요?